generators/task-runner/index.js

```diff
@@ -0,0 +1,53 @@
+const yeoman = require('yeoman-generator');
+const yosay = require('yosay');
+const _merge = require('lodash/merge');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    taskrunner() {
+      const self = this;
+
+      this.log(yosay(
+        'Configure Task Runner for Grunt'
+      ));
+      return this.prompt([{
+        type: 'checkbox',
+        name: 'selectedTaskRunner',
+        message: 'Which Task Runner do you use?',
+        choices: ['Grunt'],
+        default: ['Grunt'],
+        when: () => !self.config.get('selectedTaskRunner'),
+      }])
+      .then(answers => {
+        self.selectedTaskRunner = self.config.get('selectedTaskRunner') ||
+        answers.selectedTaskRunner;
+      });
+    },
+  },
+  writing() {
+    // Read a package.json if it exists or create it
+    const packageJSON = this.fs.readJSON(this.destinationPath('package.json'), {});
+    // Task Runner
```

Comment:
@riteshe63 You can remove this comment. This was for linters which had multiple of HTML, CSS, JS